### DISCUSSION TITLE
TriggerDagRunOperator was not creating an task instance for external Dag which have schedule interval as None

### DISCUSSION THREAD

**Discussion by krishnakanth1666** [apache/airflow#30833] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### What happened

I have four trigger dag run operators in the single Dag and trying to execute one after the other on the successful completion of the previous,but some of the task instances were not triggering and show as None State.

### How to reproduce

create two or more dags using trigger dagrunoperators and schedule them at sometime and making external dags schedule to none and see if you can find the same issue of not triggering the tasks with None status

### Operating System

Linux

### Deployment

Other

### Deployment details

deployed on gcp server 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by pankajastro** [comment]
are you facing this on airflow 2.0.0? otherwise can you please tell us the specific airflow version for example 2.5.3?

**Comment by potiuk** [comment]
Yep. This issue lacks details to act on it. It is impossible to assess if this is an airflow issue or user's mistake.

Version of Airlfow, example DAGs, logs what's happening are all needed in order to classify it as an issue with reproducible steps. Converted it into a discusson since - and more details from @krishnakanth1666 are needed to do anything about it - ideally, please create a new issue with all those details, otherwise if you can provide more details ans logs and example DAGs of yours here for discussion that could be helpful to see if you have not made any mistakes yourself and help you possibly with rectifying it..

**Reply by krishnakanth1666** [reply]
hi @potiuk ,
the version of airflow is 2.0.1

**Reply by krishnakanth1666** [reply]
![IMG-8757](https://user-images.githubusercontent.com/35325247/233956206-bae51270-98c8-436e-a12a-421f46d21421.JPG)

**Reply by krishnakanth1666** [reply]
please find the issue stated above in the screenshot

**Reply by potiuk** [reply]
PLease upgraed to latest available version first. 2.0.1 has lkely thousands of fixed bugs as it's been released more than 2 years ago. You don't even have 2.0.2 which was the lastes bugfix  release in 2.0.  It's more than likely that your problem has been fixed since (whatever it is because you have not provided logs nor example DAGs just some vagues screenshots).

You are doing everyone (including yourself) favour by upgrading first.  We are only releasing fixes to airlfow in latest released version so even if there is a bug in airflow, you will have to upgrade anyway.

Please upgrade to latest version and see if the problem persist - if it does, please submit logs and reproducible DAG examples to a  new issue.

**Reply by krishnakanth1666** [reply]
i dont have logs to produce as they weren't showing up in the UI, but i can provide you the sample dag

**Reply by potiuk** [reply]
Upgrade to latest version FIRST.

**Reply by potiuk** [reply]
Nobody is going to loose their time on looking at vague issue in 2.0.1, any more when there are 1000s of  bugs like that fixed since.

**Reply by potiuk** [reply]
(and logs are available on your machine - in case you upgrade and will have the same issue, you just need to find them - following the docs).

**Comment by krishnakanth1666** [comment]
hey logs were not been seen in the machine because of the issue stated in the above screenshot but anyways i'll try to update and see if the same issue persists but out of curiosity  is this the bug?

**Reply by potiuk** [reply]
> hey logs were not been seen in the machine because of the issue stated in the above screenshot but anyways i'll try to update and see if the same issue persists but out of curiosity is this the bug?

With 2 years and 1000s of issues, I think nobody remembers if there was a similar bug fixed since. But all release notes and issues and prs are public and if you want to be sure you can try to find out similar issue. Both released notes (you will find it in our docs) and Github history are searchable. 

Or you can upgrade and try to reproduce it and if it is fixed in latest version, your problem will be fixed. If not, then you can report it again with all the details we asked about and reproducible case.

**Comment by krishnakanth1666** [comment]
in the 2.0.1 ?

**Comment by krishnakanth1666** [comment]
![IMG-8757](https://user-images.githubusercontent.com/35325247/233963569-07fe3eca-2205-4955-a017-1da98b4cf21b.JPG)

**Comment by krishnakanth1666** [comment]
![IMG-8769](https://user-images.githubusercontent.com/35325247/233963745-15eb6512-f285-4f46-945f-3d6664ff9a6b.jpg)

**Comment by krishnakanth1666** [comment]
i feel this isn't a vague request now

**Reply by potiuk** [reply]
I feel you should upgrade to latest version and see if you can reproduce it there.

**Reply by potiuk** [reply]
Just remember that Airflow is created and maintained by 2500 contributors most of them working for free as volunteers. And you have the software for free - without any guarantees your problems will be solved. and if you are asked to spend just a little of your effort to upgrade, and testit in the latest version - this way paying back and thanks those  - mostly volunteers who already implemented thousands of fixes since the time your version has been released, rather than asking them to spend time on  investigating a vague issue that is likely was already solved, I encourage you to do so.

**Reply by krishnakanth1666** [reply]
Thanks @potiuk
